### ISSUE TITLE
tabindex="0" это не последний элемент

### ISSUE THREAD

**Issue by MaximYaskov** [javascript-tutorial/ru.javascript.info#612]
tabindex="0" делает элемент последним.
https://learn.javascript.ru/focus-blur

tabindex="0" means that the element should be focusable in sequential keyboard navigation, but its order is defined by the document's source order.
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

**Comment by iliakan** [comment]
Обновил там текст. Перечитайте, пожалуйста.

**Comment by iliakan** [comment]
@lex111

**Comment by lex111** [comment]
:+1:, тогда закрываю.

**Comment by iliakan** [comment]
Если новый текст ок =)

**Comment by MaximYaskov** [comment]
@iliakan Желательно никогда не использовать tabindex >0 потому что это формирует отдельный порядок элементов и фокус будет странно перемещаться при переходе к обычному порядку, который построил браузер на основании разметки. Надо всегда использовать tabindex='0', либо указывать tabindex >0 вообще для каждого интерактивного элемента на странице и каждый раз его перестраивать при внесении изменений.

**Comment by iliakan** [comment]
@MaximYaskov сценарии разные бывают, мы должны объяснить, как работает tabindex, такой универсальный совет многим покажется излишним.
